python/ray/train/sklearn/sklearn_predictor.py

```diff
@@ -35,6 +35,12 @@ def __init__(
         self.estimator = estimator
         super().__init__(preprocessor)
 
+    def __repr__(self):
+        return (
+            f"SklearnPredictor(estimator={self.estimator}, "
+            f"preprocessor={self._preprocessor})"
+        )
```

Comment:
what about generating these a bit more programmatically?

using: `self.__class__.__name__`
and `{name}={getattr(self, name)}`

you can clean out the prefix underscore for the name that gets printed.

This will help avoid typos

Reply:
Do you mean something like

```
arguments = []
for parameter, attribute in ("estimator", "estimator"), ("preprocessor", "_preprocessor"):
    arguments.append(f"{parameter}={getattr(self, attribute)}")
return f"{self.__class__.__name__}({', '.join(arguments)})
```

I'm not if I understand what you mean by the `{name}={repr(getattr(self, name))}` part